view/template/trial.hbs

```diff
@@ -69,8 +69,16 @@
         </div>
         <div class="col-sm-12 col-md-7 col-lg-6 col-xl-6">
             <div class="card">
-                <canvas id="trialChart" height="200%"></canvas>
-            </div>
+                  <form id = "check-compliance" method= "GET">
+                          <label>From date:-</label>
+                              <input id="fromDate"  name="fromDate" type="date" data-format = "yyyy-MM-dd" value="{{ fromDate }}" >
+                          <label>To date:-</label>
+                          <input id="toDate"  name="toDate" type="date" data-format = "yyyy-MM-dd" value="{{ toDate }}">
+                          <br><br>
+                          <input type = "submit" id = "submit" value = "Check" onclick="myFunction()"/>
```

Comment:
single space between HTML attributes and no space before or after `=`. (applies to lines 72-78)